util_test.go

```diff
@@ -20,7 +20,21 @@ func geomFromWKT(t *testing.T, wkt string) Geometry {
 func expectDeepEqual(t *testing.T, got, want interface{}) {
 	t.Helper()
 	if !reflect.DeepEqual(got, want) {
```

Comment:
What do you think about https://github.com/google/go-cmp?

Reply:
That package is pretty cool, although I want to reduce the amount of dependencies as much as possible. Since we're only using `reflect.DeepEqual` for testing, I think it's ok.